### DISCUSSION TITLE
Is DNS address accepted as endpoint or only IP?

### DISCUSSION THREAD

**Discussion by gurabli** [qdm12/gluetun#1880] [Q&A]
Hi, 
I would like to use custom provider (my own Wireguard server) to connect to, and since it is not a fix IP, i have to use the domain name. Can I do this? 

And how do I bind correctly my client wg0.conf im docker compose filel, located at /opt/gluetun/ 

ERROR reading from files: provider: server selection: wireguard: parsing peer section: endpoint host is not an IP: ParseAddr("wireguard.xyz.xyz"): unexpected character (at "wireguard.xyz.xyz")

**Comment by pvanryn** [comment]
Doesn't work (yet) ? There is an open request for this.
 [Feature request: Support hostnames on VPN_ENDPOINT_IP #788 ](https://github.com/qdm12/gluetun/issues/788)
